libraries/ESP8266WiFiMesh/src/NetworkInfo.cpp

```diff
@@ -22,6 +22,29 @@
  * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
  * THE SOFTWARE.
  */
+ 
+
+
+
+
+
+
+/********************************************************************************************
```

Comment:
I suggest marking with deprecated to get a compiler warning

Reply:
Since this class is used by other deprecated classes, adding a deprecated attribute to the class will always generate a compiler warning, even when the user is not actively using the class. I've therefore avoided adding the deprecated attribute here.